svm/src/account_loader.rs

```diff
@@ -153,65 +160,42 @@ pub fn validate_fee_payer(
 pub(crate) fn load_accounts<CB: TransactionProcessingCallback>(
     callbacks: &CB,
     txs: &[SanitizedTransaction],
-    check_results: &[TransactionCheckResult],
+    validation_results: Vec<TransactionValidationResult>,
```

Comment:
why a vec instead of slice? Only thing I could think of is to take ownership of the internal fields, but afaict `load_transaction_accounts` does not take advantage of that ownership since it clones the data.

Reply:
Yeah main thing is taking ownership of the error or the transaction details. You're right that there still is a clone in `load_transaction_accounts` for `tx_details.fee_payer_account` though, could clean that up.. but nothing else is cloned I think. But I don't really care too much either way, do you want me to change it back?

Reply:
it's a bit nicer because it matches the other args. but potentially a bit quicker if we can optimize and take the fee-payer-account value instead of clning

Reply:
OK do you mind if I remove the clone in a follow up and keep it as is?